src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/module.js

```diff
@@ -41,6 +41,22 @@ angular.module(moduleName, [])
                 }, 'customerOrderDetailWidgets');
             });
 
+            featureManager.isFeatureEnabled('UserGroupsInheritance').then(() => {
+                widgetService.registerWidget({
+                    isVisible: function (blade) { return blade.controller === 'virtoCommerce.customerModule.memberDetailController'; },
```

Comment:
Why we need this condition?

Reply:
that's how we check if we're on the needed blade

Reply:
done, removed